worlds/kh1/Options.py

```diff
@@ -0,0 +1,405 @@
+from dataclasses import dataclass
+
+from Options import NamedRange, Choice, Range, Option, Toggle, DeathLink, DefaultOnToggle, OptionSet, PerGameCommonOptions, StartInventoryPool
```

Comment:
```suggestion
from Options import NamedRange, Choice, Range, Toggle, DefaultOnToggle, PerGameCommonOptions, StartInventoryPool
```
Remove unused imports